popmon/visualization/templates/section.html

```diff
@@ -24,14 +24,13 @@ <h2>{{ section_title }}</h2>
       {%- for feature in features -%}
         <script> 
           {%- set curr = loop.index  -%}
-          var feature{{ section_index }}{{ curr }}_layout = {};
         {%- for plot_type, layout in feature.plot_type_layouts.items() -%}
-            feature{{ section_index }}{{ curr }}_layout["{{ plot_type }}"] = {{ layout | json_plot }};
+            layouts["{{ section_index }}-{{ plot_type }}"] = {{ layout | json_plot }};
```

Comment:
`layouts["{{ section_index }}-{{ plot_type }}"]`  won't work as every feature has different layout (different axis and labels). So the layout is actually being overwritten by last feature's layout ( If you see the report generated, in histograms you will see layout of TAXI_OUT instead of AIRLINE). So, the "curr" variable needs to be used.